mint/tool-versions/README.md

```diff
@@ -0,0 +1,41 @@
+# mint/tool-versions
+
+Extract the versions specified in a `.tool-versions` file, used by tools such as
+[asdf](https://asdf-vm.com/) and [mise-en-place](https://mise.jdx.dev/).
+
+Versions are available as [output values](https://www.rwx.com/docs/mint/output-values)
+after calling this leaf.
+
+## Example
+
+If your project has a `.tool-versions` file:
+
+```yaml
+nodejs 22.4.1
+ruby 3.3.2
+```
+
+Capture the versions to pass to other tasks:
+
+```yaml
+tasks:
+  - key: tool-versions
```

Comment:
We may want to have this task include `use: code`, and even add a placeholder `code` task that uses `mint/git-clone`, to make this example a little more copy-and-pasteable, especially since we're assuming there is a `.tool-versions` file that's coming from another task.